src/Pages/Employer/EmployerProfilePage.jsx

```diff
@@ -1,15 +1,52 @@
+/* eslint-disable */
 import React from 'react';
 import {
   LoggedInHeader,
   EditProfileCard,
   ReviewCard,
-  SkillsRequiredCard,
+  SkillsRequiredCards,
   Footer,
 } from '../../components';
 import {hrProfile} from '../../components/Assets';
 import {Link} from 'react-router-dom';
 
 const EmployerProfilePage = () => {
+  const handleEditJobsClick = () => {
+    
+    const jobId = localStorage.getItem('job_id');
```

Comment:
This is not where the job_id will be.

Reply:
We should create a function to edit the job by passing the id into the function.